src/main/groovy/de/jansauer/printcoverage/PrintCoverageExtension.groovy

```diff
@@ -6,9 +6,12 @@ import org.gradle.api.provider.Property
 class PrintCoverageExtension {
 
   final Property<String> coverageType
+  final Property<String> reportFile
+  final Property<String> message
 
   PrintCoverageExtension(Project project) {
     coverageType = project.objects.property(String)
-    coverageType.set('INSTRUCTION')
```

Comment:
Defaults are now handled directly in the task to allow custom task definitions with minimal configuration